cpu/stm32l1/periph/i2c.c

```diff
@@ -116,9 +116,9 @@ int i2c_init_master(i2c_t dev, i2c_speed_t speed)
         /* disable peripheral */
         i2c->CR1 &= ~I2C_CR1_PE;
         /* toggle both pins to reset analog filter */
-        _toggle_pins(port, pin_scl, pin_sda);
+        _toggle_pins(dev);
```

Comment:
@PeterKietzmann Okey, so there's a little problem. I was looking for the "right" errata sheet and found [this](http://www.st.com/web/en/resource/technical/document/errata_sheet/DM00104204.pdf). The problem is, when its copy-pasted from another code, it's maybe another errata sheet? I found nothing in chapter 2.14.7. Or I'm just blind :-P

There is something about I2C in chapter 2.4, but I'm not sure what we are searching for..